gef-ui/server/controllers/application-details/index.js

```diff
@@ -249,9 +254,12 @@ const applicationDetails = async (req, res, next) => {
       params.errors = req.errors;
     }
 
-    if (req.success) {
-      params.success = req.success;
-    }
+    // Retrieve success message from req.flash
+    const [successMessage] = req.flash('success') || ['Facility is updated'];
+
+    // Clear the flash message after retrieving it
+    req.flash('success', null);
```

Comment:
According to https://stackoverflow.com/questions/64594683/avoid-req-flash-delete-data-on-middleware retrieving it might be wiping it anyway, would we need to explicitly wipe it?

Reply:
Yes, you are right. I removed the line. Thanks, @ChristianMcc